lib/readline.js

```diff
@@ -429,7 +428,9 @@ Interface.prototype._tabComplete = function() {
           }
         }
         handleGroup(self, group, width, maxColumns);
+      }
 
+      {
```

Comment:
After googling something about "let" and "performance", the result seems to show about the `let` with `for`-loop.

@jasnell do you have some links about this considerations? In my opinion, the `let` should have better performance in non-main scope.